lib/curl/README.md

```diff
@@ -0,0 +1,65 @@
+# C API client for skycoin_rest_api/
+
+## Overview
+This API client was generated by the [OpenAPI Generator](https://openapi-generator.tech) project. By using the [OpenAPI spec](https://openapis.org) from a remote server, you can easily generate an API client.
+
+- API version: 0.25.1
+- Package version: 
+- Build package: org.openapitools.codegen.languages.CLibcurlClientCodegen
+For more information, please visit [http://127.0.0.1:6420](http://127.0.0.1:6420)
+
+## Installation
+You'll need the `curl 7.58.0` package in order to build the API. To have code formatted nicely you also need to have uncrustify version 0.67.
+
+# Prerequisites
+
+## Install the `curl 7.58.0` package with the following command on Linux.
+```bash
+sudo apt remove curl
+wget http://curl.haxx.se/download/curl-7.58.0.tar.gz
+tar -xvf curl-7.58.0.tar.gz
+cd curl-7.58.0/
+./configure
+make
+sudo make install
+```
+## Install the `uncrustify 0.67` package with the following command on Linux.
+```bash
+git clone https://github.com/uncrustify/uncrustify.git
+cd uncrustify
+mkdir build
+cd build
```

Comment:
@stdevMac The build instructions are here.
https://github.com/uncrustify/uncrustify#build

And you can find precompiled https://github.com/uncrustify/uncrustify#binaries

Reply:
i am just using exactly what creators of the generator say, maybe that's a better option, but i think is better follow their recomendations